solr/core/src/java/org/apache/solr/cli/DeleteTool.java

```diff
@@ -59,6 +59,16 @@ public String getName() {
     return "delete";
   }
 
+  @Override
+  public String getHeader() {
+    return "\nDeletes a core or collection depending on whether Solr is running in standalone (core) or SolrCloud"
+        + " mode (collection). If you're deleting a collection in SolrCloud mode, the default behavior is to also"
+        + " delete the configuration directory from Zookeeper so long as it is not being used by another collection.\n"
+        + " You can override this behavior by passing -deleteConfig false when running this command.\n"
+        + "\n"
+        + "List of options:";
```

Comment:
@janhoy  to get the formatting to work, I had to list "List of options:" here!

Reply:
Yea, guess that's how commons-cli works. We could of course append that "List of options:" string in SolrCLI so tools only need to provide the text itself...